Gemfile

```diff
@@ -16,19 +16,17 @@ gem 'rails', '~> 6.0.2', '>= 6.0.2.2'
 gem 'sass-rails', '>= 6'
 gem 'turbolinks', '~> 5'
 gem 'webpacker', '~> 4.0'
-gem 'sendgrid_actionmailer_adapter'
+# gem 'sendgrid_actionmailer_adapter'
```

Comment:
There's a few commented out lines in here. It would be great if those lines could be deleted and we could just use version history to get them back later if need be, but it's not a big deal if we're pressed for time.